scholia/scrape/ceurws.py

```diff
@@ -130,6 +130,12 @@ def tree_to_papers(tree, proceedings, proceedings_q, iso639='en'):
         paper['full_text_url'] = os.path.join(
             proceedings['url'],
             element.xpath(".//a")[0].attrib['href'])
+        
+        # Check if title_elements[0].text is None or empty
+        if title_elements[0].text is None or not title_elements[0].text.strip():
+            # Skip this paper if the title is empty
+            continue
```

Comment:
Instead of continue here, I think it would be better to more the paper['title'] in the if-block and invert the if test expression.